torch/_functorch/aot_autograd.py

```diff
@@ -31,9 +31,8 @@
 from torch._guards import TracingContext, DuplicateInputs, Source
 
 log = logging.getLogger(__name__)
-aot_forward_log = getArtifactLogger(__name__, "aot_forward_graph")
 aot_joint_log = getArtifactLogger(__name__, "aot_joint_graph")
-aot_backward_log = getArtifactLogger(__name__, "aot_backward_graph")
```

Comment:
You'll need to update the registrations here too:
https://github.com/pytorch/pytorch/blob/47f18b78ec8bbc819ccd9703e4bcd9672568eb0c/torch/_logging/_registrations.py#LL12C6-L12C6

Reply:
And the API too (I was considering opening this up to not have to update but haven't decided):
https://github.com/pytorch/pytorch/blob/47f18b78ec8bbc819ccd9703e4bcd9672568eb0c/torch/_logging/_internal.py#L129

The tests will automatically get generated so that should be it.